src/lib/PostgrestQueryBuilder.ts

```diff
@@ -17,8 +17,13 @@ export default class PostgrestQueryBuilder<T> extends PostgrestBuilder<T> {
    * Performs vertical filtering with SELECT.
    *
    * @param columns  The columns to retrieve, separated by commas.
+   * @param head  When set to true, select will void data.
+   * @param count  Count algorithm to use to count rows in a table.
    */
-  select(columns = '*'): PostgrestFilterBuilder<T> {
+  select(
+    columns = '*',
+    { head, count }: { head?: boolean; count?: 'exact' | 'planned' | 'estimated' } = {}
```

Comment:
Might be better to explicitly set the default values here:
```suggestion
    {
      head = false,
      count = null,
    }: {
      head?: boolean
      count?: null | 'exact' | 'planned' | 'estimated'
    } = {}
```

Reply:
Thanks for checking. I will commit your suggestion.